GravatarApp/Share/QRCode/VCardGenerator.swift

```diff
@@ -32,8 +47,16 @@ private func vCard(_ model: VCardModel) -> String {
     TEL:\(model.phoneNumber)
     EMAIL:\(model.email)
     URL:\(model.profileURL)
+    ADR;CHARSET=UTF-8;TYPE=HOME:;;;\(model.location);;;
```

Comment:
Good that you caught its absence. Although I noticed that the address field appears even if the location is empty 😬 It's tappable too. And it goes to Apple campus haha. Anyway I think we should remove this one if the location is empty?

<img width="320" alt="Simulator Screenshot - iPhone 16 - 2025-07-23 at 21 26 32" src="https://github.com/user-attachments/assets/9e8dd1cb-4a8c-4796-b1ea-fe7d0c572860" />